flask2postman/postman.py

```diff
@@ -0,0 +1,119 @@
+import re
+
+from .utils import trim
+
+var_re = re.compile(r"(?P<var><([a-zA-Z0-9_]+:)?(?P<var_name>[a-zA-Z0-9_]+)>)")
```

Comment:
Is it exactly the same `var_re` than for Postman v1 ? If so, please put this in `utils`. :wink: